inc/cli/class-migrate-command.php

```diff
@@ -128,7 +128,11 @@ public function wp_authors( $args, $assoc_args ) : void {
 			WP_CLI::line( sprintf( 'Processed %d posts, pausing for a breath...', $count ) );
 			sleep( 2 );
 
-			$paged++;
+			// When not overwriting, or during a dry run,
+			// need to skip pagination due to NOT EXISTS tax query.
+			if ( $dry_run || $overwrite ) {
```

Comment:
I don't think you should skip the pagination on `$overwrite`.  The query will keep returning the first page of results.

Reply:
I think the comment is wrong rather than the code. We need to paginate the results when either doing a dry run, or when overwriting, which is what the code does. The comment says the opposite 🤦

Reply:
Ha! 😄